snap/snapcraft.yaml

```
@@ -314,7 +314,9 @@ parts:
     override-prime: |
       set -eux
       for snap in "kf5-core22-sdk"; do  # List all content-snaps you're using here
-        cd "/snap/$snap/current" && find . -type f,l -not -name 'libboost_filesystem.so*' -exec rm -f "$CRAFT_PRIME/{}" "$CRAFT_PRIME/usr/{}" \;
+        cd "/snap/$snap/current" && \
+        find . -type f,l -not -path "./usr/lib/python3/dist-packages/numpy*" \
```

```suggestion
        find . -type f,l -not -path "./usr/lib/python3/dist-packages/numpy*" \
        -not -name 'libblas.so*' \
```

Add another required library to the "do not delete" blocklist: `blas`